### PR TITLE
Alternative Channel Layout

### PR DESCRIPTION
**Pull request by noslouch**
The "bespoke" layout for Indivisible.

[ticket](https://jira.wnyc.org/browse/WE-6723)

### CODE REVIEW

File: app/components/story-list/template.hbs [hunk 1]
```diff
@@ -1,7 +1,8 @@
-<ul class="list list--noborder">
+<ul class="list list--borderbottom">
```
Comment (noslouch) [inline app/components/story-list/template.hbs]: @marineb is this going to update all the other show pages. I think this needs to be conditionally applied based on the value of `flipTease`, right?

File: app/components/story-tease/template.hbs [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
 
 {{#if isFeatured}}
-  <div class="text text--small text--upper text--heavy text--spacing nudge {{if isFancyFeatured 'text--lightgray'}}">
+  <div class="text text--small text--upper text--heavy text--spacing nudge {{if isFancyFeatured 'text--white'}}">
     <p>{{status}}</p>
   </div>
 {{/if}}
 
-<div class="flag flag--rev flush">
+<div class="story-tease-meta flag {{if flipped 'story-tease--flipped flag--med'}} {{if (not flipped) 'flag--rev'}} flush">
```
Comment (noslouch) [inline app/components/story-tease/template.hbs]: this class conditional can be rewritten as `{{if flipped 'story-tease--flipped flag--med' 'fleg--rev'}}`

File: app/components/story-tease/template.hbs [hunk 3]
```diff
@@ -33,15 +51,17 @@
...
+        {{/if}}
       </div>
     </div>
 
     <div class="btn-group btn-group--storytease">
       {{#if isListenableNow}}
         {{#listen-button
           isLive=isLive
-          type=(if isFeatured 'blue-boss' 'blue-minion')
+          type = playButton
```
Comment (noslouch) [inline app/components/story-tease/template.hbs]: no spaces here

File: app/styles/_alt-channel.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  background: linear-gradient(20deg, rgba(0,0,0,0.8),rgba(0,0,0,1));
+  padding: 35px 0;
+  margin-bottom: 56px;
+
+  p {
+    margin: 0;
+  }
+}
+
+.nypr-mini-chrome .django-content .l-constrained {
```
Comment (noslouch) [inline app/styles/_alt-channel.scss]: this should go into the mini chrome add on

File: app/styles/_layout.scss [hunk 5]
```diff
@@ -108,7 +108,7 @@
  *          p Just some text in a medium width wrapper
  */
 .l-medium {
-    width: 100%;
+    width: 90%;
```
Comment (noslouch) [inline app/styles/_layout.scss]: why? the max width is already 800px

File: package.json [hunk 6]
```diff
@@ -67,7 +67,7 @@
     "ember-fetch": "1.3.0",
     "ember-getowner-polyfill": "1.0.1",
     "ember-hifi": "nypublicradio/ember-hifi#master",
-    "ember-holygrail-layout": "^0.1.3",
+    "ember-holygrail-layout": "nypublicradio/ember-holygrail-layout#master",
```
Comment (noslouch) [inline package.json]: just update this to 0.1.4

File: app/components/story-list/template.hbs (hunk 1 above)
Comment (marineb) [inline app/components/story-list/template.hbs]: yes, it's going to update the other template. Wanted! Less differences, better, no? 😃

File: app/components/story-tease/template.hbs (hunk 2 above)
Comment (marineb) [inline app/components/story-tease/template.hbs]: Oh of course. Sorry about that.

File: app/components/story-list/template.hbs (hunk 1 above)
Comment (noslouch) [inline app/components/story-list/template.hbs]: sounds good to me!

File: app/styles/_layout.scss (hunk 5 above)
Comment (marineb) [inline app/styles/_layout.scss]: Yeah. So when the screen width is => 800px, width of medium will be 800px. But it's < 800px, it becomes 90% of the screen. It's a way to handle mobile.

File: app/styles/_alt-channel.scss (hunk 4 above)
Comment (marineb) [inline app/styles/_alt-channel.scss]: Sounds good (I know it's not very elegant btw).

File: app/styles/_layout.scss (hunk 5 above)
Comment (marineb) [inline app/styles/_layout.scss]: That's been updated @noslouch!

**Pull request merged**